src/vm/prestub.cpp

```diff
@@ -240,6 +402,53 @@ void DACNotifyCompilationFinished(MethodDesc *methodDesc)
 // </TODO>
 
 
+#if defined(FEATURE_JIT_DROPPING)
+void MethodDesc::DropNativeCode()
+{
+    WRAPPER_NO_CONTRACT;
+    SUPPORTS_DAC;
+
+    g_IBCLogger.LogMethodDescAccess(this);
+
+    if (IsLCGMethod() || IsFCall())
+        return;
+
+    PCODE pCode;
+    if (HasNativeCodeSlot())
+    {
+        pCode = PCODE(NativeCodeSlot::GetValueMaybeNullAtPtr(GetAddrOfNativeCodeSlot()) & ~FIXUP_LIST_MASK);
+    }
+    else
+    {
+        if (!HasStableEntryPoint() || HasPrecode())
+            return;
+        pCode = GetStableEntryPoint();
+    }
+
+    _ASSERTE(pCode != NULL);
```

Comment:
OK, if there's really no multi-thread scenario (applies to my two previous comments as well).